TownHallTriviaApp/main/templates/main/gamePlayRoom.html

```diff
@@ -6,70 +6,59 @@
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
   </head>
   <img class="logo" src="{{ url_for('static', filename='images/banner.png') }}">  
-    <body class="body">
-    <header>
-      <div class="container">
-        <strong><nav>
-          <ul class="menu">
-            <li>
-              <form method="post" action="{{ url_for('main.playRound') }}">
-                <input type="submit" value="Round 1" />
-                <input type="hidden" name="roundId" value="Round_1">
-              </form>
-            </li>
-            <li>
-              <form method="post" action="{{ url_for('main.playRound') }}">
-                <input type="submit"value="Round 2" />
-                <input type="hidden" name="roundId" value="Round_2">
-              </form>
-            </li>
-            <li>
-              <form method="post" action="{{ url_for('main.playRound') }}">
-                <input type="submit"value="Round 3" />
-                <input type="hidden" name="roundId" value="Round_3">
-              </form>
-            </li>
-            <li>
-              <form method="post" action="{{ url_for('main.playRound') }}">
-                <input type="submit"value="Round 4" />
-                <input type="hidden" name="roundId" value="Round_4">
-              </form>
-            </li>
-            <li>
-              <form method="post" action="{{ url_for('main.playRound') }}">
-                <input type="submit"value="Round 5" />
-                <input type="hidden" name="roundId" value="Round_5">
-              </form>
-            </li>
-            <li>
-              <form method="post" action="{{ url_for('main.playRound') }}">
-                <input type="submit"value="Round 6" />
-                <input type="hidden" name="roundId" value="Round_6">
-              </form>
-            </li>
-            <li>
-              <form action="{{ url_for('main.endGame') }}">
-                <input type="submit"value="End Game" />
-              </form>
-            </li>
-          </ul>
-        </nav></strong>
-      </div>
-    </header>
+  <body class="body">
 
-    <div class="teamBanner">
-      <h1 class="teamBannerTitle">TEAM NAME: </h1>
-      <h1 class="teamBannerSeparator"> {{ teamName }}</h1>
-      <h1 class="teamBannerTitle">GAME ID: </h1>
-      <h1> {{ gameId }}</h1>
+    <div class="navMenuBanner">
+      <form method="post" class="navMenu" action="{{ url_for('main.playRound') }}">
+        <input type="submit" class="navMenu" value="ROUND 1" />
+        <input type="hidden" name="roundId" value="Round_1">
```

Comment:
i like what you did here. I'll clean this up to pass around just the round number for the round ID then you can use that for the button naming. Hows that sounds?

Reply:
sounds great! let me know or you can update it too :)